resources/js/modals/City/CreateCityModal.tsx

```diff
@@ -0,0 +1,46 @@
+import { createCity } from "~/api/cities";
+import type { ModalProps } from "~/shared.types";
+import { Button, Modal } from "~/ui";
+
+export const CreateCityModal = ({ show, onClose }: ModalProps) => {
+  async function handleSubmit(e: any) {
```

Comment:
Try to avoid using anys, you can type the submit event like this: `FormEvent<HTMLFormElement>`